docs/guides/references/migration-guide.mdx

```diff
@@ -29,40 +145,19 @@ In previous versions of Cypress, the above command would retry until the file
 existed, but would _not_ re-read it from disk if the file didn't have the
 requested property or the contents didn't match.
 
-#### `.readFile()` can no longer be overwritten with `Cypress.Commands.overwrite()`
```

Comment:
This content should still exist. Add a note at the top saying which v12 versin introduced overwriteQuery with a link to the docs